arquidiocesis-app/src/screens/parroco/DetalleParroco.js

```diff
@@ -49,7 +49,8 @@ export default (props) => {
     getParroco();
   }, []);
 
-  let getParroco = (ref) => {
+  const getParroco = (ref) => {
+    console.log(error);
```

Comment:
we could use `throw new Exception(err)` instead of just logging it into the console. 👍🏼